src/app/features/conditions/edit-conditions/conditions.component.ts

```diff
@@ -63,6 +88,42 @@ export class ConditionsComponent extends SingleRowEditorComponent<Conditions> {
     }
   }
 
+  get showInstanceInfo(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_INSTANCE_INFO;
+  }
+
+  get showLevel(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_LEVEL;
+  }
+
+  get showNearCreature(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_NEAR_CREATURE;
+  }
+
+  get ShowObjectEntryGuid(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_OBJECT_ENTRY_GUID;
+  }
+
+  get ShowRelationTo(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_RELATION_TO;
+  }
+
+  get ShowDistanceTo(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_DISTANCE_TO;
+  }
+
+  get ShowHPVal(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_HP_VAL;
+  }
+
+  get ShowHPPct(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_HP_PCT;
+  }
+
+  get ShowStandState(): boolean {
+    return this.conditionType === CONDITION_TYPES.CONDITION_STAND_STATE;
+  }
```

Comment:
@Kitzunu method names should be lowercase. E.g. `ShowStandState` -> `showStandState`

Reply:
Ahh yes that is very true. Like I did before this commit https://github.com/azerothcore/Keira3/pull/717/commits/4c1a8f1f230e3a6b3c126f211f0caa837ad4b098

I knew I made something wrong :D